qCC/ccCommandLineCommands.cpp

```diff
@@ -6292,23 +6295,27 @@ bool CommandICP::process(ccCommandLineInterface& cmd)
 				QString rotation = cmd.arguments().takeFirst().toUpper();
 				if (rotation == "XYZ")
 				{
-					transformationFilters = CCCoreLib::RegistrationTools::SKIP_NONE;
+					//keep this to not make breaking changes
 				}
 				else if (rotation == "X")
 				{
-					transformationFilters = CCCoreLib::RegistrationTools::SKIP_RYZ;
+					transformationFilters |= CCCoreLib::RegistrationTools::SKIP_RYZ;
+					cmd.print(QObject::tr("[ICP] Skip RYZ, current filter: %1").arg(transformationFilters));
```

Comment:
Is it a 'debug' log?

Reply:
it was kind of, but it is not that useless :), There were no logs about confirmations of arguments.

Reply:
Ah ok, it's just that the value of 'transformationFilters' might not be super meaningful to the users... (so maybe the second part could be 'printDebug')